sdk/src/account_info.rs

```diff
@@ -0,0 +1,99 @@
+use anchor_lang::{prelude::Result, AnchorDeserialize, AnchorSerialize};
+use solana_program::pubkey::Pubkey;
+
+use crate::{
+    account_meta::LightAccountMeta, address::PackedNewAddressParams, error::LightSdkError,
+    merkle_context::PackedMerkleContext,
+};
+
+/// Information about compressed account which is being initialized.
+#[derive(AnchorDeserialize, AnchorSerialize, Debug)]
+pub struct LightInputAccountInfo {
+    /// Lamports.
+    pub lamports: Option<u64>,
+    /// Address.
+    pub address: Option<[u8; 32]>,
+    /// Merkle tree context.
+    pub merkle_context: PackedMerkleContext,
+    /// Root index.
+    pub root_index: u16,
+}
+
+/// Information about compressed account which is being mutated.
+#[derive(AnchorDeserialize, AnchorSerialize, Debug)]
+pub struct LightAccountInfo<'a> {
+    /// Input account.
+    pub(crate) input: Option<LightInputAccountInfo>,
+    /// Owner of the account.
+    ///
+    /// Defaults to the program ID.
+    pub owner: Option<Pubkey>,
+    /// Lamports.
+    pub lamports: Option<u64>,
+    /// Account data.
+    pub data: Option<&'a [u8]>,
+    /// New Merkle tree index. Set only if you want to change the tree.
+    pub output_merkle_tree_index: Option<u8>,
```

Comment:
Here I think it should be optional, because we don't have an "output MT" when we're closing an account (provide input without expecting output).

As much as I agree with not having it optional in `new_mut` (already pushed), I would leave the Option here.